mod-circulation/src/main/resources/vega/mod-circulation/features/loans.feature

```diff
@@ -630,3 +633,70 @@ Feature: Loans tests
     And match postCheckInResponse.response.loan.status.name == 'Closed'
     And match postCheckInResponse.response.item.id == itemId
     And match postCheckInResponse.response.item.status.name == 'Available'
+
+  Scenario: When an overdue item is returned, an overdue fine is billed per the Overdue Fine Policy
+
+    * def extItemBarcode = 'FAT-1017IBC'
+    * def extUserBarcode = 'FAT-1017UBC'
+    * def extCheckInDate = '2021-11-17T13:30:46.000Z'
+    * def extDueDate = '2021-11-17T13:25:46'
+
+     # location and service point setup
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostLocation')
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostServicePoint')
+
+    # post an item
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostInstance')
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostHoldings')
+    * def postItemResponse = call read('classpath:vega/mod-circulation/features/util/initData.feature@PostItem') { extItemBarcode: #(extItemBarcode) }
+    * def itemId = postItemResponse.response.id
+
+    # post a group and user
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostGroup')
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostUser') { extUserBarcode: #(extUserBarcode) }
+
+    # post owner, manual charge and payment method
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostOwner')
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostManualCharge')
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostPaymentMethod')
+
+    # checkOut the item
+    * def checkOutResponse = call read('classpath:vega/mod-circulation/features/util/initData.feature@PostCheckOut') { extCheckOutUserBarcode: #(extUserBarcode), extCheckOutItemBarcode: #(extItemBarcode) }
+    * def loanId = checkOutResponse.response.id
+    And match checkOutResponse.response.item.id == itemId
+    And match checkOutResponse.response.item.status.name == 'Checked out'
+    And match checkOutResponse.response.loanDate == '#present'
+    And match checkOutResponse.response.dueDate contains extDueDate
```

Comment:
```suggestion
    * def checkOutResponse = call read('classpath:vega/mod-circulation/features/util/initData.feature@PostCheckOut') { extCheckOutUserBarcode: #(extUserBarcode), extCheckOutItemBarcode: #(extItemBarcode) }
    * def loanId = checkOutResponse.response.id
```
I believe we don't need to verify check-out method in this test

Reply:
fixed